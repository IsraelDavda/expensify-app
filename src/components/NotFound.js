import React from 'react';
import {Link} from 'react-router-dom';

const NotFound = () => (
    <div>
    Something Wrong this is <bold>404 Page</bold>
    <Link to="/">Go Home </Link>
    </div>
);
export default NotFound;