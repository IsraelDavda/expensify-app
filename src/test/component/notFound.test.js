import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

test('should render 404 page', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
})