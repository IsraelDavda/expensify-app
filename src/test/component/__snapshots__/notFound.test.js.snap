// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page 1`] = `
<div>
  Something Wrong this is 
  <bold>
    404 Page
  </bold>
  <Link
    to="/"
  >
    Go Home 
  </Link>
</div>
`;
